tests/test_appliancesmanager.py

```diff
@@ -0,0 +1,148 @@
+import asyncio
+
+import pytest
+
+from whirlpool.appliancesmanager import AppliancesManager
+from whirlpool.auth import Auth
+
+from .aiohttp import AiohttpClientMocker
+from .mock_backendselector import BackendSelectorMock
+from .utils import (
+    get_mock_coro,
+    mock_appliancesmanager_get_account_id_get,
+    mock_appliancesmanager_get_owned_appliances_get,
+    mock_appliancesmanager_get_shared_appliances_get,
+)
+
+BACKEND_SELECTOR_MOCK = BackendSelectorMock()
+
+
+async def test_create_headers_includes_wp_client_brand(
```

Comment:
I would prefer to avoid testing internal logic/methods of classes. Instead, I think we can have the fetch_appliances() test check that the expected headers are on the http_client_mock.mock_calls arguments.

That way we rely only on public methods + mocks to ensure behavior, making the tests less prone to break (since the public interface of the class should not change as often).

You can check the method `assert_appliance_setter_call` in tests/utils.py on how to assert using the http_client_mock